recipe/meta.yaml

```
@@ -1,20 +1,19 @@
-{% set version = "4.2.6" %}
+{% set version = "5.0.1" %}
 {% set posix = 'm2-' if win else '' %}
 {% set native = 'm2w64-' if win else '' %}
+{% set github = "https://github.com/tpq/propr" %}
 
 package:
   name: r-propr
   version: {{ version|replace("-", "_") }}
 
 source:
-  url:
-    - {{ cran_mirror }}/src/contrib/propr_{{ version }}.tar.gz
-    - {{ cran_mirror }}/src/contrib/Archive/propr/propr_{{ version }}.tar.gz
-  sha256: cf4dca710a3e6d1ab297fc1f5b0e0be85f0e36e6885da014fd099bcfef7ff099
+  url: "{{ github }}/archive/refs/tags/{{ version }}.tar.gz"
+  sha256: c0a0cf8b007501919bbe0585e412a62902f61c70cc3dfa2b56546428b16db887
 
 build:
   merge_build_host: True  # [win]
-  number: 3
+  number: 0
   rpaths:
     - lib/R/lib/
     - lib/
```

Let's take care of the Windows linking warning while we're touching this.

```suggestion
    - lib/
  missing_dso_whitelist:
    - '*/R.dll'  # [win]
```